### ISSUE TITLE
Accessibility-> When Site page displayed horizontally, "On this page" partially blocks the rest of the content from viewport

### ISSUE THREAD

**Issue by manuji** [bcgov/nr-rec-resources#658]
**Describe the bug**
When viewing the site pages on Mobile in horizontal orientation, the "On this page" section get frozen and takes up half of the viewport limiting the space to view rest of the content. This issue fails [1.3.4 Orientation (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/orientation) criterion.  

**To Reproduce**
1. Go to 'https://d1vrz2gt4nmfe5.cloudfront.net/resource/REC160773' on Mobile device
2. Use the device in horizontal orientation. 
3. Scroll down the page
4. The "On this page" section get frozen and takes up half of the viewport 

**Expected behavior**
Don't lock content to either portrait or landscape presentation.

**Screenshots**

![Image](https://github.com/user-attachments/assets/22d1a0c3-366a-4583-b350-c0df89bbb938)

**Additional context**
Tested on iPhone 16 Pro Max with Safari.
